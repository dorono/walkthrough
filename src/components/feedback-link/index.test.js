import React from 'react';
import {shallow} from 'enzyme';
import FeedbackLink from './index';

describe('FeedbackLink', () => {
    it('should render without errors', () => {
        shallow(<FeedbackLink />);
    });

    it('should match snapshot', () => {
        expect(shallow(<FeedbackLink />)).toMatchSnapshot();
    });
});
