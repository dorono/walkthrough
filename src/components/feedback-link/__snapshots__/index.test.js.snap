// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`FeedbackLink should match snapshot 1`] = `
<a
  className="root"
  target="_blank"
>
  <span
    className="question"
  >
    Have some tips to improve the explorer?
  </span>
   
  <span
    className="cta"
  >
    SEND US YOUR FEEDBACK
  </span>
</a>
`;
